src/main/java/org/cbio/gdcpipeline/util/MetaFileWriter.java

```diff
@@ -0,0 +1,48 @@
+package org.cbio.gdcpipeline.util;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.io.*;
+import java.util.Map;
+
+/**
+ * Created by Dixit on 08/07/17.
+ */
+public class MetaFileWriter {
+
+    private static String header;
+
+    private static Log LOG = LogFactory.getLog(MetaFileWriter.class);
+
+    public static void makeHeaders(Map<String, String> headers) {
```

Comment:
I wouldn't name these methods makeHeaders/writeHeaders, just call it makeMetadata/writeMetadata.